crates/ruff_linter/resources/test/fixtures/pyupgrade/UP007B.py

```diff
@@ -0,0 +1,41 @@
+import typing
```

Comment:
Can I suggest a test case of the form `List[Optional[str, int]]` ?

To be sure the rule detect also nested `Optional`. Or it is already the case? :)

Reply:
Thanks for the suggestion, I've added a test case for nested `Optional`. A  test case for nested `Union` was already there, so I didn't modify that.